### ISSUE TITLE
Permission error when running tldr 

### ISSUE THREAD

**Issue by abhitrueprogrammer** [tldr-pages/tldr-node-client#404]
Installed npm using nvm. Then installed tldr without sudo 

## Expected behavior
tldr should not throw up any error while running tldr -u 

## Actual behavior
An error when running tldr -u or tldr tar. 

## Log, debug output
Error: EACCES: permission denied, mkdir '/tmp/tldr/33329d79aee50ac92ca85fdad4dad834'

## Environment
- Operating system - macOS/Linux/Windows (add version if needed) Linux, Fedora 37
- Node.js version (`node --version`) tested in 6.14.16 & 9.4.0
- tldr-node-client version (`tldr --version`)  3.3.8

**Comment by kbdharun** [comment]
Hi, Sorry for the late reply. Do you still have this issue?

If so, can you try clearing the cache first with `tldr --clear-cache` then try updating it using `tldr --update` or `tldr -u`.

**Comment by SethFalco** [comment]
Hey hey! Thanks for reporting the issue.

I'll close this since we're unable to reproduce it and we haven't heard a follow-up from anyone. Our latest version also had a few changes and raises the minimum supported version of Node.js up to v14 which could make a difference.

Ofc you're always welcome to file another issue if this is still affecting you on the latest version.
